### ISSUE TITLE
Compile error on debian jessie with QT 5.3.2

### ISSUE THREAD

**Issue by t-b** [ola-ct/Qt-SESAM#120]
On [1](https://github.com/ola-ct/Qt-SESAM/wiki/Build-for-Linux) it is stated that QT 5.3 is okay to build with.

On my debian jessie I have

```
$qtdiag | head -n1
Qt 5.3.2 (May 19 2015, GCC 4.9.2, 64 bit, release build) on "xcb" little endian
```

but get the following compile error:

```
../../Qt-SESAM/logger.cpp: In member function ‘void Logger::log(const QString&)’:
../../Qt-SESAM/logger.cpp:74:16: error: ‘class QDebug’ has no member named ‘noquote’
       qDebug().noquote().nospace() << logMsg;
                ^
Makefile:620: recipe for target 'logger.o' failed
make[1]: *** [logger.o] Error 1
make[1]: *** Warte auf noch nicht beendete Prozesse...
make[1]: Leaving directory '/home/firma/devel/Qt-SESAM/build/Qt-SESAM'
Makefile:171: recipe for target 'sub-Qt-SESAM-make_first-ordered' failed
make: *** [sub-Qt-SESAM-make_first-ordered] Error 2
```

Something like

``` diff
$git diff .
diff --git a/Qt-SESAM/logger.cpp b/Qt-SESAM/logger.cpp
index 5338a24..eec4175 100644
--- a/Qt-SESAM/logger.cpp
+++ b/Qt-SESAM/logger.cpp
@@ -25,6 +25,7 @@
 #include <QDateTime>
 #include <QStandardPaths>
 #include <QFile>
+#include <QtGlobal>

 class LoggerPrivate {
 public:
@@ -71,7 +72,11 @@ void Logger::log(const QString &message)
       d->file.flush();
     }
     else {
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 4, 0))
       qDebug().noquote().nospace() << logMsg;
+#else
+      qDebug().nospace() << logMsg;
+#endif
     }
   }
 }
```

does the trick here, although I don't know if you want to have \@ifdef-spaghetti.

**Comment by 607011** [comment]
Thanks @t-b!

Would you like to send me a pull-request with the mentioned patch?

**Comment by t-b** [comment]
@ola-ct Yes, will do!

**Comment by t-b** [comment]
@ola-ct Done, see https://github.com/ola-ct/Qt-SESAM/pull/122.

**Comment by t-b** [comment]
@ola-ct Fixed per #122.
